packages/dotlottie-js/src/tests/utils-node.spec.ts

```diff
@@ -179,6 +181,23 @@ describe('getAnimation', () => {
 
     expect(animation).toEqual(dotLottieLottie1 as AnimationData);
   });
+
+  it('returns inlined images within the animation', async () => {
+    const animation = await getAnimation(bullAnimation, '679b6de0-4786-4239-b91e-4aa45853cc64', { inlineAssets: true });
+
+    expect(animation?.assets?.length).toEqual(5);
+    expect(JSON.stringify(animation?.assets)).toEqual(JSON.stringify(bullJson.assets));
+
+    const animation2 = await getAnimation(dotLottieAnimationWithImages, 'v2', { inlineAssets: true });
+    const animation3 = await getAnimation(dotLottieAnimationWithImages, 'v3', { inlineAssets: true });
+    const animation4 = await getAnimation(dotLottieAnimationWithImages, 'v4', { inlineAssets: true });
+    const animation5 = await getAnimation(dotLottieAnimationWithImages, 'v5', { inlineAssets: true });
+
+    expect(animation2?.assets?.length).toEqual(1);
+    expect(animation3?.assets?.length).toEqual(2);
+    expect(animation4?.assets?.length).toEqual(3);
+    expect(animation5?.assets?.length).toEqual(4);
```

Comment:
These assertions seem excessive. The primary purpose is to verify if the expected assets align with the data URL. Also, the previous assertion regarding the assets array's equality serves the same purpose.